### ISSUE TITLE
invalid world id message in chat once block is used 

### ISSUE THREAD

**Issue by 40H4** [UnRealDinnerbone/JAMD#22]
as the title would suggest once i use the block i cant seem to use it again causing me to have to use open to lan to allow myself to go into creative and replace the block maby have it reset itself to its pre use stage once it teleports you to the mining dimension

**Comment by Bass2style** [comment]
i can confirm this on the 1.18 release.

the dimension ID is correct, if you have homes or waystones in the dimension, you can still teleport.
when you break the portail (that don't drop when broken), and put another portal block, you can teleport back, until next time the block bugs.

the dimension is not affected, only the portal block.

**Comment by UnRealDinnerbone** [comment]
Hmmm I take a look at this tommorow

**Comment by Bass2style** [comment]
Hi,

since today, i totaly can't acces to the mining dimension, and it even doesn't appair on a /forge tps.

found nothing in my debug.log or the latest.log.

have a nice day.


edit : i tilted a thing, Mahou Tsoukai dimension whas not apparing in my /forge tps also. conflict anywhere with Mahou ?
edit 2 : relaunching the pack fixed it, but strange behavior. JAVD disapparait also, so forge made something weird with dimension loading.
edit 3 : never enough edits, but JAVD is also affected by the invalid wold id bug.

**Comment by ArchieBeepBoop** [comment]
I'm also having the same issue, not sure if it's linked to the NetherPortalFix (https://www.curseforge.com/minecraft/mc-mods/netherportalfix) mod that I'm using too.

**Comment by Bass2style** [comment]
this mod is also in my pack. not sure if it's related, but, wait and see.

**Comment by UnRealDinnerbone** [comment]
pushed fixed should be on curseforge soon, will need to replace portals

TLDR: the data was not saving the dim to send you to

**Comment by Bass2style** [comment]
tested out and fixed here. thank you.

can you check about the mining speed of the portal and the fact that when mined, the portal is not dropped ? (sorry, i'm too lazy to create a new report only for that little thing. XD)

have a nice day.
